templates/_linux.yaml

```diff
@@ -120,3 +120,7 @@ parameters:
 - name: SRC_PVC_NAMESPACE
   description: Namespace of the source PVC
   value: kubevirt-os-images
+- description: Randomized password
```

Comment:
Change this to something like: `Randomized password for cloud user`

Reply:
added